packages/cli/src/editor/components/OptionsPanel.tsx

```diff
@@ -59,11 +59,11 @@ const PropsEditor: React.FC<{
 	);
```

Comment:
renamed RightPanel to OptionsPanel because a left panel did not exist before this PR, and it needed to have assets and compositions, while the one on the right needed to have  renders and props, so since we were adding a new left panel, I tried decoupling the position of the panels from the functionality before we set the left-right convention... the (to be called) `LeftPanel` is now the `ExplorerPanel`